openproblems/tasks/label_projection/datasets/__init__.py

```diff
@@ -1,4 +1,6 @@
+from .cengen import cengen_batch
+from .cengen import cengen_random
 from .pancreas import pancreas_batch
 from .pancreas import pancreas_random
+from .pancreas import pancreas_random_label_noise
 from .zebrafish import zebrafish_labels
-from .zebrafish import zebrafish_random
```

Comment:
As far as I remember it was like this:
scvi methods in label projection rely on running HVGs with flavor=Seurat v3 on the dataset. This method fails (raises an error) on small datasets or low complexity datasets. Zebrafish random is small and because it randomly selects is_train portion, it often raised such error.

Reply:
I'd rather fix the method than remove the dataset, as other datasets may be low complexity in the future too. Can you catch the error and if it fails choose the HVGs a different way / not use HVGs?

Reply:
Do we know what the error was? If it's an issue of too few cells per batch for HVG selection with random sampling, then I think I'd be fine with removing the dataset. That's something that any non-random small dataset should be taking into account anyway imo.

Reply:
I think this should have the same number of cells per batch as the nonrandom dataset, but I guess because it's random there's less complexity?

Reply:
I'm not sure how less complexity leads to an error. Unless it's that there's not enough train cells per batch and cell type due to random assignment of `is_train`?

Reply:
@mxposed could you shed any light on this?

Reply:
Bumping this question